docs/contributing/introduction-to-contributing.md

```diff
@@ -17,24 +17,25 @@ keywords:
 ---
 
 Contributions are always welcome, no matter how large or small. Before contributing,
-please read the [code of conduct](./code-of-conduct.md) and follow the directions below:
+please read the [Code of Conduct](./code-of-conduct.md) and follow the directions below:
 
 ## Recommended Communication Style
 
-1. Always leave screenshots for visual changes.
-2. Always leave a detailed description in the pull request. Leave nothing ambiguous for the reviewer.
-3. Always review your code first. Run the project locally and test it before requesting a review.
-4. Always communicate in the GitHub repository. Whether in the issue or the pull request, keeping the lines of communication open and visible to everyone on the team helps everyone around you.
+- Always leave screenshots for visual changes.
+- Always leave a detailed description in the pull request. Leave nothing ambiguous for the reviewer.
+- Always review your code first. Run the project locally and test it before requesting a review.
+- Always communicate in the GitHub repository. Whether in the issue or the pull request, keeping the lines of communication open and visible to everyone on the team helps everyone around you.
```

Comment:
Refering to the code blocks in the issue, we want to only change the lines of code mentioned.
So, please revert this back to numbers.

Reply:
@HappyLife2, check this out to learn how to revert commits: https://docs.github.com/en/desktop/managing-commits/reverting-a-commit-in-github-desktop